services/management/ia/templates_service.go

```diff
@@ -159,8 +159,9 @@ func (s *TemplatesService) Collect(ctx context.Context) {
 
 	saasTemplates, err := s.downloadTemplates(ctx)
 	if err != nil {
+		// just log the error and don't return, if the user is not connected to SaaS
```

Comment:
the code is exactly the same as https://github.com/percona/pmm-managed/pull/786 which was approved

except for this deletion of return statement which was required  to solve the bug reported in this ticket https://jira.percona.com/browse/PMM-8710  it has gone through QA and is ready for merge